docs/auth.md

```diff
@@ -4,7 +4,7 @@
 
 ## Authentication Handler
 
-[`AuthHandler`](https://github.com/equip/auth/blob/master/src/AuthHandler.php) is the middleware class that coordinates the authentication process. See Equip documentation for how to [add middleware](http://equipframework.readthedocs.org/en/latest/#middleware) to an application.
+[`AuthHandler`](https://github.com/equip/auth/blob/master/src/AuthHandler.php) is the middleware class that coordinates the authentication process. See Equip documentation for how to [add middleware](/en/latest/#middleware) to an application.
```

Comment:
I removed all of the hard links and switched to using absolute relative. This prevents having to worry about protocol + domain changes.